src/config.py

```diff
@@ -21,7 +22,13 @@ class Config:
     MONGOD_CONF_DIR = f"{MONGODB_SNAP_DATA_DIR}/etc/mongod"
     MONGOD_CONF_FILE_PATH = f"{MONGOD_CONF_DIR}/mongod.conf"
     CHARM_INTERNAL_VERSION_FILE = "charm_internal_version"
-    SNAP_PACKAGES = [("charmed-mongodb", "6/edge", 118)]
+    SNAP_PACKAGES = [("charmed-mongodb", "6/edge", 121)]
```

Comment:
nice 🤩

Reply:
It made me find some bugs in the upgrade mechanism with revision upgrade ;)